v3.1/Dockerfile

```diff
@@ -41,6 +41,8 @@ ENV APACHE_RUN_USER=www-data \
     PCRE_MATCH_LIMIT_RECURSION=500000 \
     PROXY_PRESERVE_HOST=on \
     PROXY_SSL=off \
+    PROXY_SSL_VERIFY=none \
```

Comment:
I'd prefer to have `require` as the default but that would break backward compatibility as the Apache default is `none`: https://httpd.apache.org/docs/current/mod/mod_ssl.html#sslproxyverify